### PR TITLE
Forcepoint DLP Sentinel Integration

### PR DESCRIPTION
**Pull request by michaelNevinFP**
Forcepoint DLP integration into Azure Sentinel: files added logo, sample and connector json

### CODE REVIEW

File: DataConnectors/Forcepoint DLP.json [hunk 1]
```diff
@@ -0,0 +1,85 @@
+{
+    "id": "Forcepoint_DLP",
+    "title": "Forcepopint DLP",
```
Comment (preetikr) [inline DataConnectors/Forcepoint DLP.json]: Fix typo 'Forcepoint DLP'

File: DataConnectors/Forcepoint DLP.json [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+    "publisher": "Forcepoint",
+    "descriptionMarkdown": "The Forcepoint DLP connector allows you to automatically export DLP incident data from Forcepoint DLP into Azure Sentinel in real-time. This enriches visibility into user activities and data loss incidents, enables further correlation with data from Azure workloads and other feeds, and improves monitoring capability with Workbooks inside Azure Sentinel.",
```
Comment (preetikr) [inline DataConnectors/Forcepoint DLP.json]: Add the expanded form of the connector at least once at the beginning as "the Forcepoint DLP (Data Loss Prevention) connector allows ..."

File: DataConnectors/Forcepoint DLP.json [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+                    "read": true,
+                    "delete": true
+                }
+            }
+        ]
+    },
+    "instructionSteps": [
+        {
+            "title": "",
+            "description": "1. Step by step instructions to use this connector with Forcepoint DLP at this link: \n https://support.forcepoint.com/DocumentsDisplayed?version=8.7&name=Forcepoint%20DLP",
```
Comment (preetikr) [inline DataConnectors/Forcepoint DLP.json]: 1. Restructure the text in the instruction step and removed bullet as there’s just one item
"Follow step-by-step instructions provided at [Forcepoint DLP documentation](https://linkname) to configure this connector. \n"
2. Also when you are referring to product documentation links like https://support.forcepoint.com/DocumentsDisplayed?version=8.7&name=Forcepoint%20DLP , please share redirection links that you can manage on your side. That way we don’t need to re-update and re-deploy the data connector when the link changes as the redirection link in the data connectors will be static while the contents of the static link can be managed by you and updated when it changes.
3. This link https://support.forcepoint.com/DocumentsDisplayed?version=8.7&name=Forcepoint%20DLP goes to a general page and there are no instructions – is this the right link or do you plan to have detailed instructions to configure this connection in that link later – if later ETA of that? Refer to point #2 above on the link in this connector itself.

File: DataConnectors/Forcepoint DLP.json (hunk 1 above)
Comment (michaelNevinFP) [reply DataConnectors/Forcepoint DLP.json]: Fixed in latest commit

File: DataConnectors/Forcepoint DLP.json (hunk 2 above)
Comment (michaelNevinFP) [reply DataConnectors/Forcepoint DLP.json]: Fixed in latest commit

**Pull request merged**